README.md

```diff
@@ -1,2 +1,86 @@
-# flexvolume-spring-boot-starter
-Spring boot starter to read application configuration from Azure Keyvaults mounted as flexvolumes
+# HMCTS properties volume library
+
+[![Build Status](https://travis-ci.com/hmcts/properties-volume-spring-boot-starter.svg?branch=master)](https://travis-ci.com/hmcts/properties-volume-spring-boot-starter)
+[ ![Download](https://api.bintray.com/packages/hmcts/hmcts-maven/properties-volume-spring-boot-starter/images/download.svg) ](https://bintray.com/hmcts/hmcts-maven/properties-volume-spring-boot-starter/_latestVersion)
+
+This is a Spring Boot _starter_ library to read application configuration properties from files using the file name as key 
+and the file content as value. This can be used in applications that have Azure Keyvaults mounted as flexvolumes, but there 
+are probably other valid uses.
+
+
+## Usage
+Include the module into the dependency list.
+```groovy
+ compile 'uk.gov.hmcts.reform:properties-volume-spring-boot-starter:x.x.x'
+```
+
+To define which directories or files contain Azure Keyvault entries update *bootstrap.yaml* to define the related paths.
+For instance:
+```yaml
+spring:
+  application:
+    name: properties-volume-example
+  cloud:
+    propertiesvolume:
+      enabled: true
```

Comment:
I personally would leave these out fo the example configuration since they are the defaults( I probably would only make them configurable if we actually have a use case too)
I say this because it confuses the user who should really never have to use these parameters.
Add a section after the example with the more detailed options so this is not copy pasted all over the place.